couchbase_cluster_admin/cluster.py

```diff
@@ -41,6 +44,25 @@ def __init__(
         self.username = username
         self.password = password
 
+        if connect_through_ssh:
+            if not ssh_username:
+                raise ValueError("You need to specify a `ssh_username`")
+
+            self.ssh_tunnel = SshTunnel(
+                ssh_username=ssh_username,
+                remote_host=api_host,
+                remote_port=api_port,
+            )
+            self.ssh_tunnel.start()
+            self.api_host = "127.0.0.1"
```

Comment:
I guess this is always `127.0.0.1`? The api seems to expose both [local_host and local_port](https://github.com/pahaz/sshtunnel/blob/aa7342e55e49529fea020a09ba8f1eed33eb896f/sshtunnel.py#L411-L417) to obain the local endpoint. Dunno how relevant it is with non-127.0.0.1 local hosts..